addons/api/addon/serializers/fragment-managed-groups-attributes.js

```diff
@@ -0,0 +1,21 @@
+import JSONSerializer from '@ember-data/serializer/json';
+
+export default class FragmentManagedGroupsAttributesSerializer extends JSONSerializer {
+  /**
+   * If an attribute is annotated as readOnly in the model, don't serialize it.
+   * Otherwise delegate to default attribute serializer.
+   * @override
+   * @method serializeAttribute
+   * @param {Snapshot} snapshot
+   * @param {Object} json
+   * @param {String} key
+   * @param {Object} attribute
+   */
+  serializeAttribute(snapshot, json, key, attribute) {
+    const { type } = attribute;
+    let value = super.serializeAttribute(...arguments);
+    // Convert empty string to null.
+    if (type === 'string' && json[key] === '') json[key] = null;
+    return value;
```

Comment:
@gsusmi 
I end up deleting here the 'avoid serialize readOnly'. I realize on the fragment model, there is just one field, filter, and as per my understanding, that one will never be readOnly. So we do not need to contemplate a possibility we will not have.